Samples/ProjectTracker/ProjectTracker.Ui.Blazor.Server/App.razor

```diff
@@ -0,0 +1,12 @@
+<Router AppAssembly="@typeof(Program).Assembly">
+  <Found Context="routeData">
+    <AuthorizeRouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)" />
+  </Found>
+  <NotFound>
+    <CascadingAuthenticationState>
```

Comment:
The location of the CascadingAuthenticationState component surprises me here. I'd expect that to be useful in the event that a route is found, and not useful in the event that it is not found. So isn't it in the wrong template/section?

Reply:
I thought so too, but I copied this from the latest 3.1 preview templates installed this week. I agree that it seems odd, but it also appears to work...